dhall/src/builtins.rs

```diff
@@ -451,9 +451,10 @@ fn apply_builtin<'cx>(
                             );
 
                             Ret::Nir(Nir::from_kind(NirKind::TextLit(
-                                nze::nir::TextLit::new(
-                                    parts.intersperse(replacement),
-                                ),
+                                nze::nir::TextLit::new(Itertools::intersperse(
+                                    parts,
+                                    replacement,
+                                )),
```

Comment:
Why do you prefer this?

Reply:
It's more that the compiler reported an ambiguity with a method that _might_ get added to the standard library iterator trait. Reverting that commit, I see:

```
warning: a method with this name may be added to the standard library in the future
   --> dhall/src/builtins.rs:455:43
    |
455 | ...                   parts.intersperse(replacement),
    |                             ^^^^^^^^^^^
    |
    = note: `#[warn(unstable_name_collisions)]` on by default
    = warning: once this method is added to the standard library, the ambiguity may cause an error or change in behavior!
    = note: for more information, see issue #48919 <https://github.com/rust-lang/rust/issues/48919>
    = help: call with fully qualified syntax `itertools::Itertools::intersperse(...)` to keep using the current method
```

And, I basically just followed the compiler's suggestion.

Reply:
Oh I see. It's because the stdlib might add the exact same `intersperse` function, right? Feels a bit stupid but ok